tests/Brahma.FSharp.Tests/Full.fs

```diff
@@ -88,7 +88,7 @@ let FullTranslatorTests =
                     let command =
                         <@
                             fun (range:_1D) (buf:array<_>) ->
-                                let local_buf = local (Array.zeroCreate 1)
+                                let local_buf = localArray 1
```

Comment:
Is it true that only local array allocation is tested? Tests for local allocation of promotive value types also should be added.

Reply:
I have added tests for int and float types. I don't know yet about other types. But, for example, tuples now can't be local, because of the _struct_ restriction of the local function.